docs/bots.md

```diff
@@ -72,16 +72,144 @@ The content format follows the [Activity Streams 2.0 Vocabulary](https://www.w3.
+| object.id                 | The message ID of the given message on the origin server. It can be used to react or reply to the given message.                                                                                                                                                            |
+| object.name               | For normal written messages `message`, otherwise one of the known [system message identifiers](chat.md#system-messages).                                                                                                                                                    |
+| object.content            | A JSON encoded dictionary with a `message` and `parameters` key. The message can include placeholders and the [Rich Object parameters](https://github.com/nextcloud/server/blob/master/lib/public/RichObjectStrings/Definitions.php) are rendered into it in the chat view. |
+| object.mediaType          | `text/markdown` when the message should be interpreted as Markdown. Otherwise `text/plain`.                                                                                                                                                                                 |
+| object.inReplyTo          | *Optional:* 🆕 Added in Talk 21. If applicable the parent message which was quoted in this message                                                                                                                                                                          |
+| object.inReplyTo.actor    | *Optional:* 🆕 Added in Talk 21. Same data as the `actor`                                                                                                                                                                                                                   |
+| object.inReplyTo.object   | *Optional:* 🆕 Added in Talk 21. Same data as the `object` (but never contains `inReplyTo`)                                                                                                                                                                                 |
+| target.id                 | The token of the conversation in which the message was posted. It can be used to react or reply to the given message.                                                                                                                                                       |
+| target.name               | The name of the conversation in which the message was posted.                                                                                                                                                                                                               |
+
+## Receiving reaction added
+
+🆕 Added in Talk 21. Bots with the `reactions` feature receive special hooks when a reaction was added to a chat message.
```

Comment:
Is that really `reactions` instead of `reaction`? (multiple times in this PR)
https://github.com/nextcloud/spreed/blob/ce7a13a10970f80d031e38ed84ab3bc6be9c3747/lib/Model/Bot.php#L27